source/MRMeshC/MRFloatGrid.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "MRVoxelsFwd.h"
+
+MR_EXTERN_C_BEGIN
+
+/// resample this grid to fit voxelScale
```

Comment:
better comment: "resemble this grid to get cubical voxel with given size"
and similarly for the below function